src/app/.gitignore

```diff
@@ -41,3 +41,19 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# macOS
```

Comment:
Wondering if we should use each platform ignore for each section instead of adding them here, it's weird because it should also do that by default, doesn't matter too much though.

Reply:
I brought this up from another project and I think it brings more information about what they are